packages/tailwindcss/src/utilities.test.ts

```diff
@@ -2193,15 +2193,15 @@ test('border-spacing', () => {
     }
 
     .border-spacing-1 {
+      border-spacing: var(--tw-border-spacing-x) var(--tw-border-spacing-y);
       --tw-border-spacing-x: var(--spacing-1, .25rem);
       --tw-border-spacing-y: var(--spacing-1, .25rem);
-      border-spacing: var(--tw-border-spacing-x) var(--tw-border-spacing-y);
     }
```

Comment:
This does feel a bit strange to me 🤔

Reply:
I know that the order shouldn't matter here. It might be nice to see the actual properties first though, not sure,

Reply:
Introduced by: https://github.com/parcel-bundler/lightningcss/commit/d7aeff3db67ee9d15e0fefce9251cf41c0b8ec44

Reply:
Yeah weird not sure why Lightning is reordering that:

https://lightningcss.dev/playground/index.html#%7B%22minify%22%3Afalse%2C%22customMedia%22%3Atrue%2C%22cssModules%22%3Afalse%2C%22analyzeDependencies%22%3Afalse%2C%22targets%22%3A%7B%22chrome%22%3A6225920%7D%2C%22include%22%3A0%2C%22exclude%22%3A0%2C%22source%22%3A%22.foo%20%7B%5Cn%20%20--tw-border-spacing-x%3A%20var(--spacing-1%2C%20.25rem)%3B%5Cn%20%20--tw-border-spacing-y%3A%20var(--spacing-1%2C%20.25rem)%3B%5Cn%20%20border-spacing%3A%20var(--tw-border-spacing-x)%20var(--tw-border-spacing-y)%3B%5Cn%7D%22%2C%22visitorEnabled%22%3Afalse%2C%22visitor%22%3A%22%7B%5Cn%20%20Color(color)%20%7B%5Cn%20%20%20%20if%20(color.type%20%3D%3D%3D%20'rgb')%20%7B%5Cn%20%20%20%20%20%20color.g%20%3D%200%3B%5Cn%20%20%20%20%20%20return%20color%3B%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%2C%22unusedSymbols%22%3A%5B%5D%2C%22version%22%3A%22local%22%7D

Reply:
They did change the order in the linked commit, not sure if it was intentional.